src/devices/Mcp3428/Helpers.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp3428
+{
+    public enum GainEnum : byte { X1 = 0, X2 = 1, X4 = 2, X8 = 3 }
```

Comment:
since those enums are in Helpers.cs should they be internal? If not they should probably be in a separate file(s)

Reply:
They shall not be internal, they are used by the consuming software to set device configuration. 
I came up with a separate file, but I think this is just one more file that has no real purpose.